Azure/communication-ui-library-android
# [Chat] Receive read receipt on presentation layer

## Purpose
Add changes on presentation layer for receiving read receipt

## Does this introduce a breaking change?


## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull Request Checklist




## How to Test
join chat thread, send message, from other participants' side to read some message sent by local user, then verify if the correct message being marked as read.
